bindings/ocaml/src/dune

```diff
@@ -9,6 +9,7 @@
    (progn
     (chdir
      ../../../lib
+;     (run make CC=zig-cross-target-cc CXX=zig-cross-target-c++ AR=zig-cross-target-ar RANLIB=zig-cross-target-ranlib librgbmatrix.a librgbmatrix.so.1))
```

Comment:
More zig!

Reply:
This one's commented out -- if we could leave this in I'll figure out how to expose it properly it later on. But also happy to delete it to be cleaner if you prefer...